Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageBubbleView.swift

```diff
@@ -52,7 +44,7 @@ open class _ChatMessageBubbleView<ExtraData: ExtraDataTypes>: View, UIConfigProv
     
     public private(set) lazy var borderLayer = CAShapeLayer()
 
-    private var layoutConstraints: [LayoutOptions: [NSLayoutConstraint]] = [:]
+    fileprivate var layoutConstraints: [LayoutOptions: [NSLayoutConstraint]] = [:]
```

Comment:
let's make this `public` too, we will probably remove it soon, but let's keep it as flexible as we can for now.